minesweeper.py

```diff
@@ -73,27 +87,8 @@ def open_squares(self, x, y):
         else:
             if count_already_open + count_mine_locations == self.width * self.height:
                 self.game_status = GameStatus.game_won
-                statusbar_action.config(text=random.choice(win_message))
-                frames = [
-                    PhotoImage(
-                        file=where_this_file_is / "doomguy.gif",
-                        format="gif -index %i" % (i),
-                    )
-                    for i in range(8)
-                ]
-                gif_label.place(relx=0.5, rely=0.5, anchor="center")
-
-                def update(index):
-                    frame = frames[index]
-                    index += 1
-                    if index >= 8:
-                        index = 0
-                    gif_label.configure(image=frame)
-                    if self is current_game:
-                        # New game not started yet, can keep animating
-                        root.after(100, update, index)
-
-                root.after(0, update, 0)
+                statusbar_action.config(text=f"{random.choice(win_message)}")
```

Comment:
```suggestion
                statusbar_action.config(text=random.choice(win_message))
```

This seems like a merge problem, since this was already fixed.